tests/pyfunc/test_pyfunc_schema_enforcement.py

```diff
@@ -2060,21 +2075,176 @@ def predict(self, context, model_input, params=None):
             expected_example = example
         pd.testing.assert_frame_equal(loaded_example, expected_example)
 
-    # Test model serving
-    if isinstance(example, pd.DataFrame):
-        payload = {"dataframe_split": example.to_dict(orient="split")}
+    for test_example in ["saved_example", "manual_example"]:
+        if test_example == "saved_example":
+            payload = mlflow_model.get_serving_input(local_path)
+        else:
+            if isinstance(example, pd.DataFrame):
+                payload = json.dumps({"dataframe_split": example.to_dict(orient="split")})
```

Comment:
Thoughts on also testing records orient split, or do you feel that might be overkill here?

Reply:
The serving input should be a records format if example is pd.DataFrame